schemas/json/theme.json

```diff
@@ -10,7 +10,8 @@
 		"settingsProperties": {
 			"properties": {
 				"appearanceTools": {
-					"description": "Setting that enables ui tools.",
+					"description": "Setting that enables enable UI tools.",
+					"markdownDescription": "Setting that enables the following UI tools:\n\n- border: color, radius, style, width\n- color: link\n- spacing: blockGap, margin, padding\n- typography: lineHeight",
```

Comment:
I think the additional information here can be added to `description` for now. We've been sticking strictly to the draft-04 spec to ensure the schema will work with the widest variety of tools.

`markdownDescription` was added by Microsoft specifically to support Visual Studio Code (as far as I'm aware) and isn't a valid draft-04 property.

I'd like to see a little bit of research on tool usage for theme developers and on how a few more of the tools handle invalid properties before beginning to include them.

Reply:
ok, I just moved the same richer text to the description field. 👍